Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -31,7 +31,10 @@
 #include <unistd.h>
 
 HexEditor::HexEditor()
-    : m_blink_timer(Core::Timer::construct())
+    : m_blink_timer(Core::Timer::create_repeating(500, [this]() {
+        m_cursor_blink_active = !m_cursor_blink_active;
+        update();
+    }).release_value_but_fixme_should_propagate_errors())
```

Comment:
ehhhhhhhhhh...could this thing at least be a `RefPtr<>` so we can initialise it inside the ctor body instead of doing...this?